sdk/ai/Azure.AI.Projects/tests/Azure.AI.Projects.Tests.csproj

```diff
@@ -33,7 +33,7 @@
     <PackageReference Include="Azure.Monitor.OpenTelemetry.Exporter" />
     <PackageReference Include="Microsoft.ClientModel.TestFramework" />
     <PackageReference Include="System.Linq.Async" />
-    <PackageReference Include="System.Text.Json" />
+    <PackageReference Include="System.Text.Json" VersionOverride="10.0.0"/>
```

Comment:
Please don't do this.   VersionOverrides are special cases intended for short-term compatibility fixes.   We'll be updating the entire repository to the 10.x line shortly.   

Is there a reason that the tests specifically need the 10.x line before that work is complete?